### PR TITLE
draft quotas doc

### PR DESCRIPTION
**Pull request by ozzyjohnson**
Draft doc.

@dlapiduz @ramirezg @jcscottiii @mogul - Comments?

### CODE REVIEW

File: content/apps/quotas.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+
+Quotas are billed by the unit and tallied daily. A monthly bill might include 9 days at 1 unit, 11 days at 2 units and 10 days at 4 units.
+
+##### Quota Units:
+
+Each unit provides.
+
+- 10 application routes.
+- 3.75GB of RAM.
+- 1 vCPU of compute.
```
Comment (dlapiduz) [inline content/apps/quotas.md:23]: I don't think we can guarantee compute power....

File: content/apps/quotas.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+
+##### Quota Billing:
+
+Quotas are billed by the unit and tallied daily. A monthly bill might include 9 days at 1 unit, 11 days at 2 units and 10 days at 4 units.
+
+##### Quota Units:
+
+Each unit provides.
+
+- 10 application routes.
```
Comment (dlapiduz) [inline content/apps/quotas.md:21]: Is there a reason we limit the amount of routes?

Comment (ozzyjohnson) [inline content/apps/quotas.md:21]: As I see it, the number of routes should be < ∞ while > whatever the typical org will use. The current default is probably a bit low (perhaps double it), but I don't see a lot to be gained by removing the limit entirely.

File: content/apps/quotas.md (hunk 1 above)
Comment (ozzyjohnson) [inline content/apps/quotas.md:23]: Depends.

The RAM reservation extrapolates to a relative CPU share on the DEA. So there is a guarantee in that sense, but how articulate what means is something I deliberated on for a while. If we stick with M-type instances and the same unit size it remains roughly true.

We could refer to CPU in terms of share more like [Heroku](https://devcenter.heroku.com/articles/dyno-types#available-dyno-types).

Comment (ozzyjohnson) [inline content/apps/quotas.md:23]: Following up after Slack conversations this afternoon. Liking the idea of relating this as a relative share with 1 share equal to 512MB in DEA RAM/CPU and the current quota unit being a bit over 7 shares.

File: content/apps/quotas.md (hunk 2 above)
Comment (dlapiduz) [inline content/apps/quotas.md:21]: Ok we can switch it when we actually change the amount of routes... Is it ok if I merge this?

Comment (ozzyjohnson) [inline content/apps/quotas.md:21]: Good to merge by me.

**Pull request merged**